src/assets/data/defaultConfig.ts

```diff
@@ -130,6 +130,9 @@ export const DefaultSchemaSpecEndpoint = [
   DefaultSchemaSpecPath
 ].join('/')
 
+// *The order in which participant attributes are matched to protocol endpoints; format: {attributeName: orderNumber}
+export const DefaultParticipantAttributeOrder = {}
```

Comment:
Can we add human-readable-id here with order -1? I think that will by default have the highest precedence.